File: Web/RMA-Web-Shell/projects/clientcare/src/app/policy-manager/shared/entities/declaration-allowance.ts
import { AllowanceTypeEnum } from 'projects/shared-models-lib/src/lib/enums/allowance-type-enum';

export class DeclarationAllowance {
  declarationAllowanceId: number;
  declarationId: number;
  allowanceType: AllowanceTypeEnum;
  allowance: number;
}

File: Service/ClientCare/RMA.Service.ClientCare.Contracts/Entities/Member/DeclarationAllowance.cs
using RMA.Service.Admin.MasterDataManager.Contracts.Enums;

namespace RMA.Service.ClientCare.Contracts.Entities.Member
{
    public class DeclarationAllowance
    {
        public int DeclarationAllowanceId { get; set; } // DeclarationAllowanceId (Primary key)
        public int DeclarationId { get; set; } // DeclarationId
        public AllowanceTypeEnum AllowanceType { get; set; } // AllowanceTypeId
        public decimal? Allowance { get; set; } // Allowance
    }
}